src/System.Net.Http/tests/UnitTests/HttpSystemProxyTest.cs

```diff
@@ -112,7 +132,31 @@ public void HttpProxy_Local_Parsing(string bypass, int count)
                 }
                 return SuccessExitCode;
            }, bypass, count.ToString()).Dispose();
+        }
+
+        [InlineData("http://")]
+        [InlineData("http=")]
+        [InlineData("http://;")]
+        [InlineData("http=;")]
+        [InlineData("  ;  ")]
+        [InlineData("proxy.contoso.com")]
+        public void HttpProxy_InvalidSystemProxy_Null(string proxyString)
```

Comment:
This is missing either a Theory statement, so it isn't actually running.

Reply:
Fixed in the latest iteration.